### DISCUSSION TITLE
navigation group label translation - issue

### DISCUSSION THREAD

**Discussion by Windygo** [filamentphp/filament#9820] [General]
Hi

I try to translate group labels as follows:

`->navigationGroups([
            NavigationGroup::make('Management')
                ->label(fn (): string => __('navigation.management')),
])`

without success. Any idea? Thanks.

**Comment by NaungYeHtet** [comment]
+1

**Comment by ReubenSiama** [comment]
`->navigationGroups([NavigationGroup::make(__('navigation.management'))])`

Or in your resource file:
```
public static function getNavigationGroup(): ?string
{
     return __('navigation.management');
}
```

**Reply by shaheenfawzy** [reply]
That works, but now the icon isn't showing

**Reply by ReubenSiama** [reply]
I believe you will need to specify your icon in your navigation group creation.
```php
->navigationGroups([
    NavigationGroup::make(__('navigation.management'))
        ->icon('specify your icon')
])
```
